content.en/polkadot/dht/2024-23-report.md

```diff
@@ -0,0 +1,87 @@
+
+> 💡 Due to the diversity of Polkadot agents and agent versions, we've only included agents that have more than 50 peers.
+
+{{< plotly json="../../../plots/2024/06/03/polkadot-plot-churn.json" height="600px" >}}
+
+### Geolocation
+
+Geographical data is sourced from the [MaxMind database](https://www.maxmind.com), which maps IP addresses to corresponding countries.
+
+This bar plot illustrates the distribution of observed nodes across different countries.
+
+{{< plotly json="../../../plots/2024/06/03/polkadot-geo-agent-All-bars.json" height="600px" >}}
+
+This plot displays the weekly geographical distribution of nodes, categorized by country.
+
+{{< plotly json="../../../plots/2024/06/03/polkadot-geo-agent-All-lines.json" height="600px" >}}
+
+### Cloud Providers
+
+Cloud providers data is sourced from [Udger](https://udger.com/resources/datacenter-list), which maps IP addresses to known hosting providers.
+
+#### Cloud Hosting Rate
+
+This line chart displays the count of nodes within the Polkadot network that are hosted on known commercial cloud providers, compared to those that are not. It tracks the distribution over a specified period, offering insights into the infrastructure preferences for node hosting.
+
+Regular analysis of this chart can reveal trends in the adoption of cloud services for nodes. Such information is crucial for understanding the network's resilience and the potential reliance on cloud infrastructure.
+
+{{< plotly json="../../../plots/2024/06/03/polkadot-cloud-rate-agent-All-lines.json" height="600px" >}}
+
+This bar chart presents the weekly distribution of Polkadot nodes among various cloud providers, including nodes not hosted within data centers (grouped under _Non-Datacenter_).
+
+{{< plotly json="../../../plots/2024/06/03/polkadot-cloud-agent-All-bars.json" height="600px" >}}
+
+The line chart illustrates the trends in the distribution of all Polkadot nodes across cloud providers over the given time period. Note that nodes hosted outside of data centers are not included in this representation.
+
+{{< plotly json="../../../plots/2024/06/03/polkadot-cloud-agent-All-lines.json" height="600px" >}}
+
+### Crawls
+
+#### Protocols
+
+This plot illustrates the evolving count of nodes supporting each of the listed protocols over time. It exclusively presents data gathered from nodes accessible through a libp2p connection via our crawler. The identification of supported protocols relies on the [libp2p identify protocol](https://github.com/libp2p/specs/tree/master/identify), hence necessitating a libp2p connection for discovery.
+
+{{< plotly json="../../../plots/2024/06/03/polkadot-crawl-protocols.json" height="1200px" >}}
+
+#### Errors
+
+{{< plotly json="../../../plots/2024/06/03/polkadot-connection-errors-single.json" height="600px" >}}
+
+{{< plotly json="../../../plots/2024/06/03/polkadot-crawl-errors-single.json" height="600px" >}}
+
+## Stale Node Records
+
+### All Peers
+
```

Comment:
```suggestion
```

There are no other categories to be shown (such as mainnet on Ethereum reports), so no need to specify "All Peers".